.scalafmt.config

```
@@ -0,0 +1,20 @@
+maxColumn = 120
+align = most
+continuationIndent.defnSite = 2
+assumeStandardLibraryStripMargin = true
+docstrings = ScalaDoc
+lineEndings = preserve
+includeCurlyBraceInSelectChains = false
+danglingParentheses = true
+
+align.tokens = [
+  {
+    code = ":"
+  }
+]
+
+newlines.alwaysBeforeCurlyBraceLambdaParams  = false
```

Not sure what this character is in here?

```suggestion
newlines.alwaysBeforeCurlyBraceLambdaParams = false
```